docs/components.rst

```diff
@@ -45,9 +45,9 @@ Well, in Discord, clicking buttons and using slash commands are called ``interac
 Responding to interactions
 __________________________
 
-When responding, you have 3 choices in how you handle interactions. You can either wait for them in the command itself, or listen for them in a global event handler (similar to :func:`on_slash_command_error`), or register async function as callback callback.
+When responding, you have 3 choices in how you handle interactions. You can either wait for them in the command itself, or listen for them in a global event handler (similar to :func:`on_slash_command_error`), or register async function as component callback.
 
-Lets go through the most common method first, responding in the event itself. We simply need to :func:`wait_for` the event, just like you do for reactions. For this we're going to use :func:`wait_for_component() <discord_slash.utils.manage_components>`, and we're going to only wait for events from the action row we just sent.
+Lets go through the most common method first, responding in the command itself. We simply need to :func:`wait_for` the event, just like you do for reactions. For this we're going to use :func:`wait_for_component() <discord_slash.utils.manage_components>`, and we're going to only wait for events from the action row we just sent.
```

Comment:
Why don't we split them up into sections for organizational purposes, that way we can extend upon them later if we want to further update/modify these methods of responses?